pane.go

```diff
@@ -24,7 +24,8 @@ type Pane struct {
 
 // Creates a new pane object.
 func NewPane(id int, sessionId int, sessionName string, windowId int,
-	windowName string, windowIndex int, active bool) *Pane {
```

Comment:
I'm wondering why it was formatted that way. But `go-fmt` says it is correct.

Reply:
I use neovim with "gofumpt", maybe if deviates from `go fmt`